keyboards/omnikeyish/keymaps/default/keymap.c

```diff
@@ -0,0 +1,18 @@
+#include "quantum.h"
```

Comment:
This should be: 

```suggestion
#include QMK_KEYBOARD_H
```

And the rest of the includes can be removed.

Reply:
'#include QMK_KEYBOARD_H' breaks the build for me on macos

Compiling: keyboards/omnikeyish/keymaps/default/keymap.c                                           <command-line>: error: empty filename in #include
keyboards/omnikeyish/keymaps/default/keymap.c:6:24: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'keymaps'

Reply:
Yeah, you need to have a `omnikeyish.h` file file.  And it needs to have `quantum.h` included in it.  If you don't, it will error out.  

If you do, and you still run into this issue, let me know.